cmd/maya-apiserver/cstor-operator/spc/controller.go

```diff
@@ -178,9 +178,8 @@ func (c *Controller) updateSpc(oldSpc, newSpc interface{}) {
 		return
 	}
 	// Enqueue spc only when there is a pending pool to be created.
-	if c.isPoolPending(spc) {
-		c.enqueueSpc(newSpc)
-	}
+	c.enqueueSpc(newSpc)
```

Comment:
:+1: good find

Reply:
would be good if we can add a comment like
`// Enqueue'ing in no-pending-pool case also to handle any miss in delete timestamps to remove finalizer on SPC/BDCs

Reply:
done